### PR TITLE
Form HTML & CSS

### PR DESCRIPTION
**Pull request by mallowmew**

### CODE REVIEW

File: form-style.css [hunk 1]
```diff
@@ -0,0 +1,131 @@
+.form-container {
+  padding: 12px;
+  width: 100%;
+  background: #393432;
+}
+
+.form-container h1 {
+  margin: auto;
```
Comment (MotNosniktaw) [inline form-style.css]: use a different level h instead of conditional formatting it?

File: form-style.css [hunk 2]
```diff
@@ -0,0 +1,131 @@
+.form-container {
+  padding: 12px;
+  width: 100%;
```
Comment (MotNosniktaw) [inline form-style.css]: has no effect?

File: form-style.css [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+  color: #e8e8e8;
+}
+
+.errors {
+  text-align: center;
+}
+
+@media screen and (min-width: 600px) {
+  .new-game-form {
+    position: relative;
```
Comment (MotNosniktaw) [inline form-style.css]: is this position relative here for a reason

File: form-style.css (hunk 1 above)
Comment (mallowmew) [reply form-style.css]: In face-to-face convo you mentioned that this can affect SEO - will ask Sam.

**Pull request merged**